pkg/webhook/clusterresourceplacement/v1beta1_clusterresourceplacement_validating_webhook.go

```diff
@@ -42,6 +42,10 @@ func (v *clusterResourcePlacementValidator) Handle(_ context.Context, req admiss
 			klog.ErrorS(err, "failed to decode v1beta1 CRP object for create/update operation", "userName", req.UserInfo.Username, "groups", req.UserInfo.Groups)
 			return admission.Errored(http.StatusBadRequest, err)
 		}
+		if err := validator.ValidateClusterResourcePlacement(&crp); err != nil {
+			klog.V(2).InfoS("v1beta1 cluster resource placement has invalid fields, request is denied", "operation", req.Operation, "namespacedName", types.NamespacedName{Name: crp.Name})
+			return admission.Denied(err.Error())
+		}
```

Comment:
I didn't get why we need to move this up

Reply:
This ensures that we validate the CRP regardless of the operation CREATE/UPDATE and then if it's an UPDATE we check to see if the update is valid